test/source/test.ts

```diff
@@ -66,8 +67,9 @@ const testWithBrowser = (
   return async (t: AvaContext) => {
     let closeMockApi: (() => Promise<void>) | undefined;
     if (isMock) {
-      const mockApi = await startMockApiAndCopyBuild(t);
-      closeMockApi = mockApi.close;
+      t.mockApi = await startMockApiAndCopyBuild(t);
+      t.mockApi.attesterConfig = ATTESTER_ACCOUNT_INIT_SETUP_MOCK_CONFIG;
```

Comment:
I would rather that we don't set it at all by default, and it gets set explicitly in tests where it's used.

If it's too much work at this stage to tailor the settings on each test, then that could be a follow-up PR. But This line should be somewhere in each `test(` every time it's needed, instead of here.

Reply:
The `ATTESTER_ACCOUNT_INIT_SETUP_MOCK_CONFIG` configuration is useful/needed for setting up mock data for common accounts when we pass the `acct` variable to the `testWithBrowser` function, as the variable name suggests.

Reply:
This way we will have tons of this duplicated code in each test.
Is this fine?

`await BrowserRecipe.setUpCommonAcct(t, browser, acct);`

Reply:
Could be a follow-up PR, it's a larger change across the tests - just adding `const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);` into every test that needs it. Laborious but worth it imo, it will be consistent and explicit. And as you can see, more flexible for configuration. Attester is probably not the only config that will run into this issue, most mocks probably will. They will be needed during setup.

You actually wouldn't need to do this either yet:

```diff
const testWithBrowser = (
- acct: CommonAcct | undefined,
  cb: (t: AvaContext, browser: BrowserHandle) => Promise<void>,
  flag?: 'FAILING'
): Implementation<unknown[]> => {
```

Maybe just add a comment to `acct` that we plan to remove it. Then start passing `undefined` instead?

```diff
  test(
      'compose - expired can still send',
-     testWithBrowser('ci.tests.gmail', async (t, browser) => {
+     testWithBrowser(undefined, async (t, browser) => {
+     const acctEmail = '...';
        const expiredEmail = 'expired.on.attester@domain.com';
        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
        t.mockApi!.attesterConfig = {
          pubkeyLookup: {
+          // add required attester config here
            [expiredEmail]: {
              pubkey: expiredPubkey,
            },
          },
        };
+      const settingsPage = await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);
       const composePage = await ComposePageRecipe.openStandalone(t, browser, 'compose');
```

Up to you what you think will be less work. Can either rip the bandaid all at once (remove the `acct` argument and add `openSettingsLoginApprove` to all that used it), or do it gradually with `undefined`.

Reply:
Ok, maybe can remove the `acct` arg and add `await BrowserRecipe.setUpCommonAcct(t, browser, acct);`.

I think single `await BrowserRecipe.openSettingsLoginApprove(t, browser, acctEmail);` line isn't enough. Need to set passphrase depends on account.
Those all logics are already in `setUpCommonAcct` and I think we can just call `BrowserRecipe.setUpCommonAcct`

Reply:
Right.

Reply:
Can of course confirm with @sosnovsky in case he had some other idea or plans

Reply:
Removing `acct` parameter and running `await BrowserRecipe.setUpCommonAcct(t, browser, acct)` when needed sounds like a good choice for me 👍